src/types/models/index.ts

```diff
@@ -129,6 +129,14 @@ export interface MessageLead {
   }
 }
 
+export interface LeasingInterest {
```

Comment:
We can send a leasing interest without any information about the customer?

Reply:
We can send it to the backend and it would not pass validation. We have optional values to support the empty form.

Reply:
Ahh makes sense